test/test_configs/config.sqs.test.yml

```diff
@@ -1,5 +1,6 @@
 mocks:
-  sqs:
-    component: sqs
+  - name: sns_sqs
```

Comment:
i think we can remove this complete file and let the sqs test use the sns_sqs test config

Reply:
✅